frontend/views/pages/Contributions.vue

```diff
@@ -11,6 +11,7 @@ page(pageTestName='contributionsPage' pageTestHeaderName='contributionsTitle' v-
         v-for='(contribution, index) in fakeStore.receiving.nonMonetary'
         :key='`contribution-${index}`'
       )
+        //- TODO: verify this is safe (no XSS)
```

Comment:
what do you mean by `XSS`? probably isn't safe because it's a text written by the user (in an input), so we could write there anything to be processed by `v-html`.

Reply:
What I mean is if this string contains any input that was generated by the user, we cannot use `v-html`, and I didn't take the time to verify whether that was the case.